docs/modules/videoprocessor.html

```diff
@@ -319,8 +319,62 @@ <h3>Custom processor usage during meeting preview</h3>
 		<div class="col-4 col-menu menu-sticky-wrap menu-highlight">
 			<nav class="tsd-navigation primary">
 				<ul>
-					<li class="globals  ">
-						<a href="../globals.html"><em>Globals</em></a>
+					<li class=" ">
+						<a href="../modules.html">Exports</a>
```

Comment:
seems like global is now a module, does this affect the URL?

Reply:
It only affects this single page: https://aws.github.io/amazon-chime-sdk-js/globals.html, it will become `https://aws.github.io/amazon-chime-sdk-js/modules.html`. For rest of the guide, class, interface, the URL remains the same.

Reply:
Can you add a custom globals.html to generated docs folder and mention to check modules.html instead? Cause this means its breaking for documentation, and would be good to let builders know to go to right place.

Reply:
Is it easier to just rename modules.html to globals.html in our post generation script?

Reply:
I think we can do that. The git change is actually to rename the `globals.html` to `modules.html`. We can revert this for sure.

Reply:
oh wait, basically all `.html` files have a link to `modules.html`, we also need to replace these links.

Reply:
Alright, I think the best solution is to duplicate the `modules.html` and rename it to `global.html`. So we keep `globals.html` and `modules.html` at the same time. So we don't need to update those links, and they can still link to `modules.html`.

I will go ahead with this solution. Let me know if you have any concern.